cs-rin-ru-enhanced-mod.user.js

```diff
@@ -412,12 +425,13 @@ function tagify() {
     if (options.custom_tags) {
         $(".titles, .topictitle").each(function () {
             const titleElem = this;
+            const parentElem = titleElem.parentElement
```

Comment:
is it needed anywhere else? if not then maybe it would be better to just move it to colorize function